### PR TITLE
Updated link about authentication file

### PR DESCRIPTION
**Pull request by millette**

### CODE REVIEW

File: src/get-default-auth-cfg.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ module.exports = async () => {
   let migrated = false
 
   const config = {
-    _: 'This is your Now credentials file. DON\'T SHARE! More: https://git.io/v5ECz',
+    _: 'This is your Now credentials file. DON\'T SHARE! More: https://zeit.co/docs/features/configuration#now.json',
```
Comment (TooTallNate) [inline src/get-default-auth-cfg.js]: This actually isn't correct. The link you are updating it to is about your project local `now.json` file, _not_ the `~/.now/auth.json` file.

Comment (millette) [reply src/get-default-auth-cfg.js]: I see. I searched the README history to find an anchor for ```global-configuration``` (where the git.io leads to) to no avail. Maybe you can point to the right place?

Comment (leo) [reply src/get-default-auth-cfg.js]: https://goo.gl/mbf4CZ

**Comment by leo** [comment]
Thank you!

**Pull request merged**